polkadb/memoryDB.go

```diff
@@ -85,8 +85,9 @@ func (db *MemDatabase) Del(key []byte) error {
 }
 
 // Close ...
-func (db *MemDatabase) Close() {
+func (db *MemDatabase) Close() bool {
 	// do nothing
+	return true
```

Comment:
Why is this needed? Also -- why doesn't this function do anything?

Reply:
@ansermino given its the `memDB` I don't think it's needed at all. Will remove.

Reply:
@ansermino actually we do need it to fulfill the function signatures requirement for `Database` interface. The reason it does not do anything is because its a mapping and there is nothing to close. We could clear the mapping here I suppose.